src/transformers/onnx/features.py

```diff
@@ -552,9 +553,61 @@ def get_model_class_for_feature(feature: str, framework: str = "pt") -> Type:
             )
         return task_to_automodel[task]
 
+    @staticmethod
+    def determine_framework(model: str, framework: str) -> str:
+        """
+        Determines the framework to use for the export.
```

Comment:
Added a unit test in [b832988](https://github.com/huggingface/transformers/pull/18615/commits/b83298803227cf8a79c2e0eb5a4724cf52275513), but I put it under `tests/onnx/test_onnx_v2::OnnxUtilsTestCaseV2`. I just noticed you specified `test_features.py`, but it does not exist yet. I can create it if you'd like, or should I leave it as is?

Reply:
Thank you!! Yes, please create a new `test_features.py` file for this test (we usually map `transformers/path/to/module.py` with `tests/path/to/test_module.py`)

Reply:
Thanks! Done in [67416f2](https://github.com/huggingface/transformers/pull/18615/commits/67416f22094bb29b7bdcfe22854ee00ef4e1e112).

In [8da5990](https://github.com/huggingface/transformers/pull/18615/commits/8da5990fda344f7eb754bc8a9a0f44fac56b00fd) and  I registered the tests in `utils/tests_fetcher.py` because of a failure I got in CI saying that the test would not be discovered. Is this the correct way to add them?

In [63198fd](https://github.com/huggingface/transformers/pull/18615/commits/63198fd2b7c018cb2b20580cf1ec8614079e078f) I added `tf` for for the pip install steps for `run_tests_onnxruntime` and `run_tests_onnxruntime_all` in `.circleci/config.yml` so that `TFAutoModel` can be used. Also added `-rA` flags so that the results would be more verbose. In the logs for `run_tests_onnxruntime` it can be seen that the new unit tests are tested.

Reply:
Thank you for registering the test - this is indeed the way to include it :)